File: C-to-C# Port Wizard/assign3Samples/fileOut.c
/*
 * fileOut.c
 *
 * a simple source file demonstrating file output
 */

#include <stdio.h>
#include <stdlib.h>

int main (void)
{
	FILE *fp;

	fp = fopen ("testOutput.txt", "w");
	if (fp == NULL) 
	{
		printf ("**Error: Unable to open file : testOutput.txt for writing\n");
		return 1;
	}

	fprintf (fp, "hello world\n");
	fclose (fp);

	printf("File testOutput.txt has been created\n");

	return 0;
}

File: C-to-C# Port Wizard/PortWizard/PortWizard/bin/Debug/assign3Samples/fileOut.cs
/*
* fileOut.c
*
* a simple source file demonstrating file output
*/

using System;
using System.IO;
using System.Text;


namespace Something
{
	class Program
	{
		static void Main()
		{
			FileStream fp;
			
			fp = File.Open ("testOutput.txt", FileMode.OpenOrCreate, FileAccess.Write);
			if (fp == null) 
			{
				Console.WriteLine ("**Error: Unable to open file : testOutput.txt for writing");
				return;
			}
			
			fp.Write(Encoding.ASCII.GetBytes( "hello world\n"), 0, 12);
			fp.Close();
			
			Console.WriteLine("File testOutput.txt has been created");
			
			return;
		}

	}
}
